adam/curriculum/phase1_curriculum.py

```diff
@@ -1326,6 +1364,72 @@ def _make_sit_curriculum() -> Phase1InstanceGroup:
     )
 
 
+def make_take_grab_template(
+    agent: TemplateObjectVariable,
+    theme: TemplateObjectVariable,
+    *,
+    use_adverbial_path_modifier: bool,
+    spatial_properties: Iterable[OntologyNode] = None,
+) -> Phase1SituationTemplate:
+    # X puts Y on Z
+    return Phase1SituationTemplate(
+        "take",
```

Comment:
nit: prefer - Future spots to look at marked with 🗒️ 
```suggestion
        f"{agent.handle}-take-{theme.handle}",
```